corehq/apps/domain/templates/domain/partials/license_explanations.html

```diff
@@ -1,22 +1,31 @@
 {% load i18n %}
 <a data-toggle="modal" href="#license-explanations">More information about licenses...</a>
-<div class="modal hide fade" id="license-explanations">
-    <div class="modal-header">
-        <a class="close" href="#" data-dismiss="modal">&times;</a>
-        <h3>About available licenses for shared media and projects</h3>
-    </div>
+<div class="modal fade" id="license-explanations">
     <div class="modal-body">
```

Comment:
this should be `modal-dialog` see https://docs.google.com/document/d/1jOIrCZlrmECfzo-iyAYjeVFqBv7ilnXjdgDWjGLmqd4/edit#heading=h.8hrj8xjboyf4